qiita_db/reference.py

```diff
@@ -6,15 +6,71 @@
 # The full license is in the file LICENSE, distributed with this software.
 # -----------------------------------------------------------------------------
 from __future__ import division
-from os.path import join
+from os.path import join, dirname, basename, exists
+from shutil import move
+from glob import glob
+import warnings
+import re
 
 from .base import QiitaObject
-from .exceptions import QiitaDBDuplicateError
+from .exceptions import QiitaDBDuplicateError, QiitaDBWarning, QiitaDBError
 from .util import (insert_filepaths, convert_to_id,
                    get_mountpoint)
 from .sql_connection import SQLConnectionHandler
 
 
+def _rename_sortmerna_indexed_db_files(smr_db, db_name, db_version):
+    """Renames the SortMeRNA indexed db files to use a controlled name
+
+    Parameters
+    ----------
+    smr_db : str
+        The base path to the sortmerna_indexed_db
+    db_name : str
+        The name of the reference database
+    db_version : str
+        The version of the reference database
+
+    Raises
+    ------
+    QiitaDBError
+        If smr_db does not contain a correct SortMeRNA indexed db
+    """
+    smr_db_dir = dirname(smr_db)
+
+    # Rename the .stats file
+    old_fp = "%s.stats" % smr_db
+    if not exists(old_fp):
+        raise QiitaDBError(
+            "%s does not look like a correct SortMeRNA indexed database."
+            % smr_db)
+
+    new_fp = join(smr_db_dir, "%s_%s.stats" % (db_name, db_version))
+    move(old_fp, new_fp)
+
+    suff = ["pos", "bursttrie", "kmer"]
+
+    for s in suff:
+        g_re = "%s.%s_[0-9]*.dat" % (smr_db, s)
+        s_re = "(?<=%s_)[0-9]*(?=.dat)" % s
```

Comment:
Thanks for the tips ~~regex whisperer~~ @adamrp 
@wasade I think is possible that multiple files are produced. It will be a point that if the reference DB grows enough, multiple files will be generated. In any case, I'd prefer to not limit the system, and just support everything just in case. Better be safe than sorry.